config.cfg

```
@@ -0,0 +1,211 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Non-ideal compressible fluid flow in a converging-         %
+%                   diverging supersonic nozzle for siloxane fluid MM          %
+% Author: Alessio Secchiaroli                                                  %
+% Institution: TU Delft                                                        %
+% Date: 10/03/2025                                                             %
```

Please correct the date to 18/03/2025

```suggestion
% Date: 18/03/2025                                                             %
```